optaplanner-quickstart/pom.xml

```diff
@@ -11,12 +11,12 @@
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
     <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-    <optaplanner-quarkus.version>7.43.0.Final</optaplanner-quarkus.version>
+    <optaplanner-quarkus.version>8.0.0.Final</optaplanner-quarkus.version>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
     <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
     <docker-plugin.version>0.28.0</docker-plugin.version>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Is this acceptable?

OptaPlanner 8.0.0.Final.jar has java 11 compatible class files. It targets 11. It doesn't work on Java 8.

Reply:
OptaPlanner 8.0.0.Final now works on Java 8 too.
 Is it fine that this quickstarts defaults to 11 to stop 11 from proliferating?